src/stories/common/TableFiltering.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import TableFiltering from '~common/Extensions/TableFiltering/TableFiltering';
```

Comment:
And a blank line between import packages and other imports.

Also, please correct the import:
`import TableFiltering from '~common/Extensions/TableFiltering/';`